nupic/research/frameworks/pytorch/models/le_sparse_net.py

```diff
@@ -153,7 +153,11 @@ def add_sparse_linear_layer(
             )
         else:
             network.add_module(
-                "linear{}".format(suffix), SparseWeights(linear, weight_sparsity)
+                "linear{}".format(suffix),
```

Comment:
I don't think this change is correct. Here, the parameter is defined as "Pct of weights that are allowed to be non-zero". You would need to use `sparsity = 1 - weight_sparsity`

Reply:
Thanks for catching that, fixed.